### PR TITLE
Upload spec local fs

### PR DESCRIPTION
**Pull request by fhoering**

### CODE REVIEW

File: cluster_pack/filesystem.py [hunk 1]
```diff
@@ -48,10 +51,39 @@ def _expose_methods(child_class: Any, base_class: Any, ignored: List[str] = []):
         setattr(child_class, method_name, _make_function(base_class, method_name))
 
 
+def _chmod(self, path, mode):
+    os.chmod(path, mode)
+
+
+def _rm(self, path, recursive=False):
+    print(locals())
```
Comment (t-henri) [inline cluster_pack/filesystem.py:59]: Can you remove the print please

File: cluster_pack/uploader.py [hunk 2]
```diff
@@ -124,14 +124,24 @@ def upload_spec(
     force_upload: bool = False,
     fs_args: Dict[str, Any] = {}
 ) -> str:
+    """Upload an environment from a spec file
+
+    :param spec_file: the spec file, must be requirements.txt or conda .yaml
```
Comment (t-henri) [inline cluster_pack/uploader.py:129]: extra space after conda

File: cluster_pack/uploader.py [hunk 3]
```diff
@@ -124,14 +124,24 @@ def upload_spec(
     force_upload: bool = False,
     fs_args: Dict[str, Any] = {}
 ) -> str:
+    """Upload an environment from a spec file
+
+    :param spec_file: the spec file, must be requirements.txt or conda .yaml
+    :param package_path: the path where to upload the package
+    :param force_upload: wether the cache shoud lbe cleared
```
Comment (t-henri) [inline cluster_pack/uploader.py:131]: whether and should be

File: cluster_pack/filesystem.py (hunk 1 above)
Comment (fhoering) [reply cluster_pack/filesystem.py]: Done

File: cluster_pack/uploader.py (hunk 2 above)
Comment (fhoering) [reply cluster_pack/uploader.py]: Done

File: cluster_pack/uploader.py (hunk 3 above)
Comment (fhoering) [reply cluster_pack/uploader.py]: Done

**Pull request merged**